ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -58,11 +64,28 @@ type structure_route = {
     Presence of ':' in any of the values indicates this is IPv6 related.
 }
 type structure_rule = {
+    @{to selector}
+    "to" ? network_ip_cmd_prefix
+    @{from selector}
+    "from" ? network_ip_cmd_prefix
+    @{not action (false value means no not action; also the default when not is not defined)}
```

Comment:
Uhm... I'm having a hard time parsing this... What does it do?

Reply:
?

Reply:
@stdweird?

Reply:
it's the inverse of a selector (not sure why i thought it was the inverse of an action) (`man ip rule`). i guess you want me to change it :wink:

Reply:
I want to understand it, which I guess means yes, I do want you to change it :stuck_out_tongue: